src/langchain_google_alloydb_pg/model_manager.py

```diff
@@ -196,6 +201,7 @@ async def __aget_model(self, model_id: str) -> Optional[AlloyDBModel]:
                 model_qualified_name VARCHAR,
                 model_auth_type google_ml.auth_type,
                 model_auth_id VARCHAR,
+                header_gen_fn VARCHAR,
```

Comment:
Seems like its not possible to return either params `header_gen_fn` or `generate_headers_fn`, once its set. [ref](go/mem_model-info-view) 
Looks intentional coz it could leak creds.

Reply:
```suggestion
```